launcher/image/test/scripts/test_gpu_unsupported_gputype.sh

```diff
@@ -0,0 +1,17 @@
+ #!/bin/bash
+set -euo pipefail
+source util/read_serial.sh
+
+# This test requires the workload to run and printing
+# corresponding messages to the serial console.
+SERIAL_OUTPUT=$(read_serial $1 $2) 
+print_serial=false
+
+if echo $SERIAL_OUTPUT | grep -q 'unsupported gpu type'
```

Comment:
could you refactor your script to accept another param to check if the expected message exists in serial logs? That way you don't need three scripts with same logics.

Reply:
along expected messages, there are different success and failure messages so I didn't parameterized it and kept it separate.

Reply:
then could you put those three separate files into a new directory something like `scripts/gpu` since they're highly related to gpu tests.